frontend/src/components/ Footer/footer.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import logo from './imagenes/logo.png';
+import redes from './imagenes/redes.png';
+import binamp from './imagenes/logobinamp.png';
+import trazado from './imagenes/Trazado.png';
+import copyryght from './imagenes/copyryght.png';
+
+class Fooster extends React.Component {
```

Comment:
Cuidado que dice **Fooster** en vez de **Footer**

Reply:
Además, como no  están usando manejando el estado del componente ni usando alguna de las funciones del ciclo de vida de un componente de clase es preferible que este pase a ser un componente funcional, así no se cargan métodos que no se van a usar.
```suggestion
function Footer() {
```